google/ggrc-core
# GGRC-6472 Prevent adding custom roles on those objects on admin page (Control)

# Issue description
Prevent add/update/delete custom roles on Control object on admin page. Only actions from GGRCQ side have permissions to create/update custom roles for Control, special header should be added to requests

# Steps to test the changes

Log in as Administrator
Try to add/update/delete custom role to Control
Expected result: Response with "Method not allowed" and status code = 405 should be send
Run integration tests

# Solution description
Add ReadOnlyGGRC mixin to Control model
Update event hooks (add/update/delete) with GGRCQ header validation

# Sanity checklist




# PR Review checklist
